src/mon/PGMap.cc

```diff
@@ -3778,7 +3782,8 @@ int process_pg_map_command(
       } else {
         int filter = pg_string_state(state_str);
         if (filter < 0) {
-          *ss << "'" << state_str << "' is not a valid pg state";
+          *ss << "'" << state_str << "' is not a valid pg state,"
+              << " available choices: " << pg_state_string(-1);
```

Comment:
@branch-predictor Please refer to https://github.com/ceph/ceph/pull/16465, by suggestions from @liewegas  we drop the client-side validations of pg-states now, so user will lose any pg_state info.

This patch tries to be slightly helpful due to the reason above.